HotTips/Engine/TipManager.cs

```diff
@@ -24,7 +24,14 @@ public List<GroupOfTips>[] GetPrioritizedTipGroups()
 
         public TipInfo GetTipInfo(string globalTipId)
         {
-            return _allTips?[globalTipId];
+            if (!string.IsNullOrEmpty(globalTipId) && _allTips.ContainsKey(globalTipId))
```

Comment:
Is this change necessary? Wouldn't it return null if globalTipId is null or if allTips does not contain the tipId?